opencv/src/main/java/org/bytedeco/javacpp/helper/opencv_core.java

```diff
@@ -1829,6 +1843,45 @@ public static double cvNorm(CvArr arr1, CvArr arr2) {
         @Override public int arrayStep() { return step(0); }
 
         public static final Mat EMPTY = null;
+
+        @Override public <I extends Indexer> I createIndexer(boolean direct) {
+            BytePointer ptr = arrayData();
+            int size = arraySize();
+            int dims = dims();
+            int depth = depth();
+            int elemSize = CV_ELEM_SIZE1(depth);
```

Comment:
I think we could just call `elemSize1()` here instead?

Reply:
Ok, I didn't spot this method. It'll trigger a JNI call but I guess we don't care here.
CV_ELEM_SIZE1 definition can be deleted then.